packages/components/psammead-social-embed/src/index.test.jsx

```diff
@@ -1,47 +1,64 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, waitFor } from '@testing-library/react';
```

Comment:
I have a feeling that `waitFor` is deprecated - can we use `act` instead? We did this in Simorgh recently https://github.com/bbc/simorgh/pull/8405

Reply:
I _think_ `waitFor` is okay but `wait` is deprecated: https://testing-library.com/docs/dom-testing-library/api-async/#wait-deprecated-use-waitfor-instead